### PR TITLE
Fix crash for feature ID texture without channels property

### PR DESCRIPTION
**Pull request by lilleyse**
In `EXT_mesh_features` the `channels` property is no longer required and defaults to `[0]`. Updates the code to reflect that and avoid a crash. The same fix was required for `EXT_structural_metadata`.

* https://github.com/CesiumGS/glTF/blob/e3d043eae175b2bc746b5dcbbd8f1c7a63f90090/extensions/2.0/Vendor/EXT_mesh_features/schema/featureIdTexture.schema.json#L20-L22
* https://github.com/CesiumGS/glTF/blob/e3d043eae175b2bc746b5dcbbd8f1c7a63f90090/extensions/2.0/Vendor/EXT_structural_metadata/schema/propertyTexture.property.schema.json#L20-L22

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
```
Comment (ptrgags) [inline CHANGES.md:1]: seems like you might need to run `prettier`, not sure why the pre-commit hook didn't format files correctly.

**Review by ptrgags — COMMENTED** [review]
@lilleyse I had a couple comments

File: Source/Scene/GltfLoader.js [hunk 2]
```diff
@@ -1121,7 +1121,7 @@ function loadFeatureIdTexture(
   // Though the new channel index is more future-proof, this implementation
   // only supports RGBA textures. At least for now, the string representation
   // is more useful for generating shader code.
-  const channelString = textureInfo.channels
+  const channelString = defaultValue(textureInfo.channels, [0])
```
[the remark this answers is not in the document]
Comment (lilleyse) [reply Source/Scene/GltfLoader.js]: My reasoning for using `defaultValue` was that it was a one-time cost. But I'm fine with the approach you suggested too.

**Comment by lilleyse** [comment]
@ptrgags updated

**Comment by ptrgags** [comment]
Looks good, thanks @lilleyse!

**Pull request merged**